queries.sql

```diff
@@ -1,10 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * FROM animals WHERE name LIKE '%mon';
-SELECT * FROM animals WHERE EXTRACT(YEAR FROM date_of_birth) BETWEEN 2016 AND 2019;
-SELECT * FROM animals WHERE neutered = TRUE AND escape_attempts < 3;
-SELECT * FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
-SELECT * FROM animals WHERE weight_kg > 10.5;
-SELECT * FROM animals WHERE neutered = TRUE;
-SELECT * FROM animals WHERE NOT (name = 'Gabumon');
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT name FROM animals WHERE name LIKE '%mon';
+SELECT date_of_birth FROM animals WHERE EXTRACT(YEAR FROM date_of_birth) BETWEEN 2016 AND 2019;
```

Comment:
- [ ] In the project description, here is what is required of us
![Screenshot from 2023-05-02 08-08-36](https://user-images.githubusercontent.com/46388113/235612233-49ec178c-d58c-4c85-914c-2f19a316f246.png)
and not `date_of_birth`